manager.go

```diff
@@ -231,6 +259,8 @@ func (m *Manager) CloseAll() {
 	m.connMu.Lock()
 	defer m.connMu.Unlock()
 
+	m.netAppender.Close()
```

Comment:
thanks for catch -- ran into this while removing netappender and did not think about close case

zooming out, would be good to have 'can it start and stop cleanly without network connection' as a CI test case

this is done in 21379349

tested that it can close with wifi disconnected: netappender takes like 20 seconds to fully exit (that's ticketed [here](https://viam.atlassian.net/browse/RSDK-7866)), but no crash
```
2024-06-24T20:19:46.828Z	INFO	netlogger	logging/net_appender.go:233	error logging to network: rpc error: code = DeadlineExceeded desc = context deadline exceeded
2024-06-24T20:19:56.834Z	INFO	netlogger	logging/net_appender.go:233	error logging to network: rpc error: code = DeadlineExceeded desc = context deadline exceeded
2024-06-24T20:20:01.838Z	INFO	netlogger	logging/net_appender.go:233	error logging to network: rpc error: code = DeadlineExceeded desc = context deadline exceeded
```